### PR TITLE
Feat: Notes filter pages

### PR DESCRIPTION
**Pull request by ramalingaa**
Added Notes feature for video
 - User can now add/delete notes for a video
Added filter chips on Video listing page
 - User can toggle between categories

### CODE REVIEW

**Review by Rohanmond — APPROVED** [review]
Liked your file structure and code is looking good to merge

File: src/backend/controllers/NotesController.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    new Response(
+      404,
+      {},
+      {
+        errors: ["The email you entered is not Registered. Not Found error"],
+      }
+    );
+  }
+  // const userNotes = schema.users.findBy({ _id: user }).notes;
+  return new Response(200, {}, { notes: user.notes });
```
Comment (lomesshh) [inline src/backend/controllers/NotesController.js]: remove commented line

File: src/backend/controllers/NotesController.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  if (user) {
+    const { video } = JSON.parse(request.requestBody);
+    if (user.notes.some((item) => item._id === video._id)) {
+     
+      return new Response(
+        409,
+        {},
+        {
+          errors: ["The video is already in your liked videos"],
+        }
```
Comment (lomesshh) [inline src/backend/controllers/NotesController.js]: give proper error message to note controller

**Review by lomesshh — APPROVED** [review]
UI looks preety cool

do suggested changes
rest all looks good to me

File: src/backend/controllers/NotesController.js (hunk 2 above)
Comment (ramalingaa) [reply src/backend/controllers/NotesController.js]: Changes made

**Pull request merged**